### ISSUE TITLE
Why is the output result of rerank not a number between 0 and 1?

### ISSUE THREAD

**Issue by houjibofa2050** [FlagOpen/FlagEmbedding#108]
from FlagEmbedding import FlagReranker
reranker = FlagReranker('BAAI/bge-reranker-base', use_fp16=False) #use fp16 can speed up computing

score = reranker.compute_score(['query', 'passage'])
print(score)

scores = reranker.compute_score([['what is panda?', 'hi'], ['what is panda?', 'The giant panda (Ailuropoda melanoleuca), sometimes called a panda bear or simply panda, is a bear species endemic to China.']])
print(scores)

The results are as follows
0.4474879801273346
[-8.15439224243164, 6.182111740112305]

What is the range of score?

**Comment by staoxiao** [comment]
Hi, the reranker is optimized based on cross-entropy loss, so the relevance score is not bounded to a specific range.

**Comment by satyaborg** [comment]
Use a sigmoid function to get bounded probabilities between [0,1] from the raw logits:
```
def sigmoid(score: float) -> float:
    return 1 / (1 + np.exp(-score))
```
